person.rb

```diff
@@ -0,0 +1,26 @@
+require('securerandom')
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission = nil)
```

Comment:
- [ ] Hi @wacoo, please let's check the previous reviewer comment once more. He even use a screenshot from the requirements showing that the default value for the parent_permission parameter is `true` and not `nil`. Please let's adhere to the requirements.